### ISSUE TITLE
Move this repo to dcos org

### ISSUE THREAD

**Issue by rimusz** [bernadinm/tf_dcos_core#17]
Would make sense to move this repo to `dcos` org and name it `terraform_dcos_core` :)

**Comment by bernadinm** [comment]
Agreed! I will move this over. We might need to keep the name of the project as `tf_dcos_core` or at a minimum keep `tf` to follow the naming connection of the community modules here: https://github.com/terraform-community-modules

**Comment by a-nldisr** [comment]
Cannot find this under the dcos org, has it been moved? If i want to commit support for 1.11.0, do i have to do it here?

**Comment by cprovencher** [comment]
@rimusz @a-nldisr 
moved just now. 
https://github.com/dcos/tf_dcos_core
